assets/app/view/game/round/stock.rb

```diff
@@ -406,18 +405,17 @@ def render_select_par_slot
           h(:div, children)
         end
 
-        def render_take_loan
-          take_loan = lambda do
-            process_action(Engine::Action::TakeLoan.new(@current_entity, loan: nil))
+        def render_game_buttons
+          buttons = []
+          @step.game_buttons(@current_entity).each_with_index do |game_button, index|
+            lam = -> { game_button_pushed(index) }
+            buttons << [h(:button, { on: { click: lam } }, game_button[:description])]
           end
-          [h(:button, { on: { click: take_loan } }, "Take Loan (#{@game.format_currency(@game.loan_amount)})")]
```

Comment:
I believe render contract is that it return html elements. So: `h(:div, buttons)`

Reply:
No longer needed - this code was reviewed